src/libui_sdl/libui/examples/CMakeLists.txt

```diff
@@ -13,26 +13,26 @@ macro(_add_example _name)
 	endif()
 endmacro()
 
-_add_example(controlgallery
-	controlgallery/main.c
-	${_EXAMPLE_RESOURCES_RC}
-)
+# _add_example(controlgallery
+# 	controlgallery/main.c
+# 	${_EXAMPLE_RESOURCES_RC}
+# )
 
-_add_example(histogram
-	histogram/main.c
-	${_EXAMPLE_RESOURCES_RC}
-)
+# _add_example(histogram
+# 	histogram/main.c
+# 	${_EXAMPLE_RESOURCES_RC}
+# )
 
-_add_example(cpp-multithread
-	cpp-multithread/main.cpp
-	${_EXAMPLE_RESOURCES_RC}
-)
+# _add_example(cpp-multithread
+# 	cpp-multithread/main.cpp
+# 	${_EXAMPLE_RESOURCES_RC}
+# )
 if(NOT WIN32)
 	target_link_libraries(cpp-multithread pthread)
 endif()
 
-add_custom_target(examples
-	DEPENDS
-		controlgallery
-		histogram
-		cpp-multithread)
+# add_custom_target(examples
+# 	DEPENDS
+# 		controlgallery
+# 		histogram
+# 		cpp-multithread)
```

Comment:
There are some broken function calls in this example code.

This "example" code doesn't appear necessary for functionality. Just commenting it out is a huge hack, but I think it might be acceptable here.